Moli/Core/Screens/Report/ReportView.swift

```diff
@@ -2,12 +2,51 @@ import SwiftUI
 
 struct ReportView: View {
     var user: API.User.UserMinified
+    var data = ReportViewModel()
+    @State var successfulReportSent = false
+
+    func handleReportPress(item: ReportItem) {
+        Task {
+            do {
+                try await API.User.reportUser(userId: user.userId, reason: item.reason, description: item.description)
+                self.successfulReportSent = true
+            } catch {
+                GlobalSnackbar.shared.showSnackbar(
+                    .init(text: "Something went wrong, could not report user", variant: .danger)
+                )
+            }
+        }
+    }
 
     var body: some View {
         VStack {
-            Text("Report")
+            if successfulReportSent {
+                ReportFinalView()
+            } else {
+                List {
```

Comment:
:bomb: Ser att enligt designen ska det finnas en header här
```suggestion
                List {
                    Section(header: Text("Select a reason")) {
```